### DISCUSSION TITLE
GPS Speed video start repeat feature works when device is in dry environment but not when in wet environment

### DISCUSSION THREAD

**Discussion by breflick** [gopro/labs#465] [General]
Device: GoPro Hero11 Black

Repeat feature for GPS Speed video start works when device is in a vehicle or boat (i.e. dry) but does not work when on water ski (i.e. wet) meaning that GPS Speed video start does work initially on water ski but does not repeat GPS Speed start after stop speed is reached causing video recording to stop.

**Comment by dnewman-gpsw** [comment]
Are you using this workaround https://github.com/gopro/labs/issues/446 ?

**Reply by breflick** [reply]
I am not because the issue is not that video doesn't stop.  The issue is that while repeat feature for GPS Speed feature is selected,  it only repeats when device is dry.  When device is wet, GPS speed feature works at selected start and stop speeds for initial run on water ski ; however, further runs on water ski are not captured.  To understand, I ski to one end of the lake then stop.  I then ski back to the other end of the lake then stop.  Again, device works following this pattern when in the boat but not when on the water ski.

**Reply by dnewman-gpsw** [reply]
It is not very logical that wet has anything to do with it, if the camera remains above the water, that is why setting the fix for repeat speed trigger is a good idea. However, if the camera is submersed, there is no GPS signal.  Is that the scenario?

**Comment by breflick** [comment]
All runs start with the device in the same location of being a few inches below the top of the water so lack of GPS signal for the initial run is clearly not an issue.  I will try increasing stop speed setting in case the video is stopping due to lack of GPS signal because of device being more than a few inches below water line and not due to reaching the stop speed as ski sinks back into water after initial run.

**Reply by breflick** [reply]
Increasing the start and stop speed solved the issue.
